retrofit2/build.gradle

```diff
@@ -0,0 +1,10 @@
+dependencies {
+    compile project(':core')
+    compile 'com.google.guava:guava'
+    compile 'com.squareup.retrofit2:retrofit'
+
+    testCompile 'junit:junit'
+    testCompile 'com.squareup.retrofit2:adapter-guava'
+    testCompile 'com.squareup.retrofit2:converter-jackson'
+    testCompile 'org.assertj:assertj-core'
```

Comment:
You don't need to add junit and assertj-core because it's added by `../build.gradle`.